projects/slap_manipulation/src/slap_manipulation/utils/h5_utils.py

```diff
@@ -0,0 +1,41 @@
+import h5py
+import rospy
+from geometry_msgs.msg import TransformStamped
+from matplotlib import pyplot as plt
+from tf2_ros import tf2_ros
+
+from home_robot.utils.data_tools.image import img_from_bytes
+
+
+def view_keyframe_imgs(file_object: h5py.File, trial_name: str):
```

Comment:
should these go in data_tools instead of here?

Reply:
Sure, I am okay with that. I will just move the entire file to `home_robot/utils/data_tools` folder